### PR TITLE
remove external sources and replace font with a submodule

### LINKED ISSUE #6: CDNs entfernen
https://github.com/chaos-jetzt/website_pelican/blob/7f2c116a74e9f993bf0f3da8b9432e40a94af7b7/clean-blog/templates/base.html#L55-L65

_Originally posted by @YtvwlD in https://github.com/chaos-jetzt/website_pelican/pull/4_

### PR DESCRIPTION
**Pull request by n0emis**
Closes #6

### CODE REVIEW

File: clean-blog/templates/base.html [hunk 1]
```diff
@@ -137,23 +135,12 @@
             <div class="row">
                 <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
                     <p class="copyright text-muted">All content <a href="https://creativecommons.org/licenses/by-sa/4.0/">CC by SA</a> unless stated otherwise.</p>
-                    <p class="copyright text-muted">Blog powered by <a href="http://getpelican.com">Pelican</a>,
```
Comment (e1mo) [inline clean-blog/templates/base.html:140]: I'm wondering why we are removing the credits to the tools we are using?

Comment (n0emis) [reply clean-blog/templates/base.html:140]: oops, that was not on purpose...

File: clean-blog/templates/base.html [hunk 2]
```diff
@@ -137,23 +135,13 @@
...
-                which takes great advantage of <a href="http://python.org">Python</a>.</p>
```
Comment (e1mo) [inline clean-blog/templates/base.html:141]: Oh, that one is also missing

**Comment by YtvwlD** [comment]
This also removes local JavaScript files, but just from the HTML and not from the repository. (I'm okay with removing them, but that should probably be noted somewhere.)

Also, there are still `@import`s in the CSS if I'm not mistaken.

**Comment by labcode-de** [comment]
> Also, there are still `@import`s in the CSS if I'm not mistaken.
Yes, there is still one import of Google Fonts in clean_blog/static/css/bootstrap.cyborg(.min).css, which should be removed

**Comment by n0emis** [comment]
> This also removes local JavaScript files, but just from the HTML and not from the repository. (I'm okay with removing them, but that should probably be noted somewhere.)
> 
> Also, there are still `@import`s in the CSS if I'm not mistaken.

jep, sorry. i put it in my local branchname but forgot to mention it here. :woman_facepalming: 

the remote imports are now gone too...

File: clean-blog/templates/base.html [hunk 3]
```diff
@@ -59,16 +59,7 @@
...
-        <link href='http://fonts.googleapis.com/css?family=Lora:400,700,400italic,700italic' rel='stylesheet' type='text/css'>
-        <link href='http://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800' rel='stylesheet' type='text/css'>
-
-        <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
-        <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
-        <!--[if lt IE 9]>
-            <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
-            <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
-        <![endif]-->
+        <link href="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/fonts/open-sans.css" rel="stylesheet">
```
Comment (labcode-de) [inline clean-blog/templates/base.html:62]: My browser cannot find the file. Shouldn't the line be changed as follows?
```html
<link href="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/fonts/open-sans/open-sans.css" rel="stylesheet">
```

Comment (n0emis) [reply clean-blog/templates/base.html]: yes, indeed

**Pull request merged**